tests/test_storage.py

```diff
@@ -267,6 +268,140 @@ def test_get_orders(storage_mock):
     assert orders[0].order_id == ORDER_ID
 
 
+@pytest.mark.parametrize(
+    "params, expected_output",
+    [
+        (
+            {
+                "workspace_orders": True,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": None,
+                "name": None,
+            },
+            JSON_ORDERS,
+        ),
+        (
+            {
+                "workspace_orders": True,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": ["CREATED"],
+                "name": None,
+            },
+            JSON_ORDERS,
+        ),
+        (
+            {
+                "workspace_orders": True,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": ["TEST"],
+                "name": None,
+            },
+            JSON_ORDERS,
+        ),
+        (
+            {
+                "workspace_orders": True,
+                "return_json": False,
+                "order_type": "TASKING",
+                "statuses": ["CREATED"],
+                "name": None,
+            },
+            [
+                {
+                    "id": "ddb207c0-3b7f-4186-bc0b-c033f0d2f32b",
+                    "userId": "1094497b-11d8-4fb8-9d6a-5e24a88aa825",
+                    "workspaceId": "workspace_id_123",
+                    "dataProvider": "OneAtlas",
+                    "status": "FULFILLED",
+                    "createdAt": "2021-01-18T16:18:16.105851Z",
+                    "updatedAt": "2021-01-18T16:21:31.966805Z",
+                    "assets": ["363f89c1-3586-4b14-9a49-03a890c3b593"],
+                    "createdBy": {
+                        "id": "1094497b-11d8-4fb8-9d6a-5e24a88aa825",
+                        "type": "USER",
+                    },
+                    "updatedBy": {"id": "system", "type": "INTERNAL"},
+                },
+            ],
+        ),
+        (
+            {
+                "workspace_orders": True,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": None,
+                "name": "Testing Name",
+            },
+            JSON_ORDERS,
+        ),
+        (
+            {
+                "workspace_orders": False,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": None,
+                "name": None,
+            },
+            JSON_ORDERS,
+        ),
+        (
+            {
+                "workspace_orders": False,
+                "return_json": True,
+                "order_type": "TASKING",
+                "statuses": None,
+                "name": None,
+            },
+            JSON_ORDERS,
```

Comment:
Expected output is still the same in the majority of tests, we will need to adjust this in the future.